com.valvesoftware.Steam.json

```diff
@@ -20,9 +20,12 @@
         "--allow=multiarch",
         "--allow=devel",
         "--persist=.",
-        "--extension=org.freedesktop.Platform.Compat32=directory=lib/32bit",
-        "--extension=org.freedesktop.Platform.Compat32=version=1.6",
-        "--extension=org.freedesktop.Platform.GL32=directory=lib/32bit/lib/GL",
+        "--extension=org.freedesktop.Platform.Compat.i386=directory=lib/i386-linux-gnu",
+        "--extension=org.freedesktop.Platform.Compat.i386=version=unstable",
```

Comment:
Is this intentionally unstable?

Reply:
That is a mistake. Nice you saw it. I had unstable also installed.